codegen/templates/resourceobject2.tmpl

```diff
@@ -0,0 +1,62 @@
+//
+// Code generated by grafana-app-sdk. DO NOT EDIT.
+//
+
+package {{.Package}}{{$root := .}}
+
+import (
+    "github.com/grafana/grafana-app-sdk/resource"
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+type {{.ObjectTypeName}} struct {
+    metav1.TypeMeta `json:",inline"`
+    metav1.ObjectMeta `json:"metadata"`
```

Comment:
I'm in favour of having ObjectMeta vs Metadata, better have less generated code 👍🏼